File: Android/sources/com/my/tracker/obfuscated/p.java
package com.my.tracker.obfuscated;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.content.IntentFilter;

public final class p {
    int a = 1;
    int b = -1;

    public o a(Context context) {
        Intent registerReceiver = context.registerReceiver((BroadcastReceiver) null, new IntentFilter("android.intent.action.BATTERY_CHANGED"));
        if (registerReceiver == null) {
            return null;
        }
        int intExtra = registerReceiver.getIntExtra("status", 1);
        int i = -1;
        int intExtra2 = registerReceiver.getIntExtra("level", -1);
        int intExtra3 = registerReceiver.getIntExtra("scale", -1);
        if (intExtra2 >= 0 && intExtra3 > 0) {
            i = (intExtra2 * 100) / intExtra3;
        }
        return new o(intExtra, i);
    }
}

File: IOS/v2/com/my/tracker/obfuscated/p.swift
package com.my.tracker.obfuscated;.swift
import iOS.content.BroadcastReceiver;.swift
import iOS.content.Context;.swift
import iOS.content.Intent;.swift
import iOS.content.IntentFilter;.swift
public final class p {.swift
int a = 1;.swift
int b = -1;.swift
public o a(Context context) {.swift
Intent registerReceiver = context.registerReceiver((BroadcastReceiver) null, new IntentFilter("iOS.intent.action.BATTERY_CHANGED"));.swift
if (registerReceiver == null) {.swift
return null;.swift
}.swift
int intExtra = registerReceiver.getIntExtra("status", 1);.swift
int i = -1;.swift
int intExtra2 = registerReceiver.getIntExtra("level", -1);.swift
int intExtra3 = registerReceiver.getIntExtra("scale", -1);.swift
if (intExtra2 >= 0 && intExtra3 > 0) {.swift
i = (intExtra2 * 100) / intExtra3;.swift
}.swift
return new o(intExtra, i);.swift
}.swift
}.swift
